packages/sdk/src/oracle/store/read.ts

```diff
@@ -1,24 +1,25 @@
-import assert from "assert";
 import filter from "lodash/filter";
 
 import type {
   State,
   Chain,
-  Inputs,
-  Request,
+  InputRequest,
   Erc20Props,
   ChainConfig,
   Context,
   Memory,
   User,
   RequestIndexes,
+  RequestIndex,
   OptimisticOracleEvent,
+  FullRequest,
 } from "../types/state";
 import type { JsonRpcSigner, BigNumber, Provider } from "../types/ethers";
 import { TransactionConfirmer, requestId } from "../utils";
 import { OptimisticOracle } from "../services/optimisticOracle";
 import { Erc20 } from "../services/erc20";
 import { SortedRequests } from "../services/sortedRequests";
+import { assertExists as assert } from "../errors";
```

Comment:
Similar to below, I think renaming this is very confusing for the reader, as the normal assert does something different.